src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.swift

```diff
@@ -227,11 +228,19 @@ final class OnboardingInfoViewController: UIViewController {
 		stateStateLabel.text = exposureNotificationsEnabled ? onboardingInfo.stateActivated : onboardingInfo.stateDeactivated
 
 		guard !pageSetupDone else {
```

Comment:
It's getting hard to understand the overall logic.
In generell I think we should, if possible, separate creating the UI and updating the UI to different funcs.

I would propose a quick fix for better readable. Use "if else" instead of "guard". What do you think?

if pageSetupDone {
 ....
} else {
 ....
}

pageSetupDone = true

Reply:
refactored the code. Introduced new method pageSetup()